toolkit/tools/pkg/imagecustomizerlib/testdata/nochange-config.yaml

```diff
@@ -1 +1,5 @@
 os:
```

Comment:
`nochange-config.yaml` is intended to be completely empty. If you need something different, then add a new config file.